python-sdk/docs/development/RELEASE.md

```diff
@@ -35,7 +35,7 @@ branch from main (please use `git cherry-pick -x <commit id>` to retain
 git commit hashes and messages). Depending on the fix there might be some git conflicts to resolve. If you run into conflicts, please
 resolve said conflicts, run `git add .`, and then `git cherry-pick --continue` to continue the merge.
 
-After cherry-picking all needed fixes, follow the instructions to [create a release from the release branch](https://github.com/astronomer/astro-sdk/blob/main/python-sdk/docs/development/RELEASE.md#Creating a release from the release branch)
+After cherry-picking all needed fixes, follow the instructions to [create a release from the release branch](https://github.com/astronomer/astro-sdk/blob/main/python-sdk/docs/development/RELEASE.md#creating-a-release-from-the-release-branc).
```

Comment:
```suggestion
After cherry-picking all needed fixes, follow the instructions to [create a release from the release branch](https://github.com/astronomer/astro-sdk/blob/main/python-sdk/docs/development/RELEASE.md#creating-a-release-from-the-release-branch).
```

Did you mean `branc` actually? Somehow it is not taking to right section, but `branch` takes it there if that's where we want to point it to.